client/src/components/web-extension/Common/Grid/index.jsx

```diff
@@ -93,7 +102,15 @@ const Grid = ({ data = [], metadata = {}, onRowClick, className, isLoading }) =>
 				);
 			})}
 			{isLoading && <Bar />}
-		</div>
+		</>
+	);
+
+	return isInfiniteScroll ? (
+		<InfiniteScroll className={`cd_we_grid ${className || ''}`} {...infiniteScrollProps}>
+			{renderContent()}
+		</InfiniteScroll>
+	) : (
+		<div className={`cd_we_grid ${className || ''}`}>{renderContent()}</div>
```

Comment:
This can be:
```
import cn from "classnames"

const className = cn("cd_we_grid", className);

return isInfiniteScroll ? (
<InfiniteScroll className={className} {...infiniteScrollProps}>
	{renderContent()}
</InfiniteScroll>
) : (
<div className={className}>{renderContent()}</div>
)

```